Portfolio2.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <title>Elmi Farah's Portfolio'</title>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" type="text/css " href="portfolio2.css ">
+   </head>
+   <body>
+      <header>
+         <div id="bannertop">
+            <div class="menue">
+               <ul>
+                  <li class="logo"> <img src="maydenacademylogo.png"/> </li>
+               	  <li class="active"> ABOUT ME </li> 
+                  <li class="activ"> PORTFOLIO </li> 
+                  <li class="acti"> CONTACT INFO </li> 
+               </ul>
+     	  </div>
+ 	 </div>
+      </header>
+      <div id="main">
+         <div id="headertwo">
+            <div>
```

Comment:
This div isn't doing anything, you can remove it so it's just the headertwo div  with the h1s inside

Reply:
so the div id main one correct

Reply:
Yeah but I think you still have one more div than you need. The `<div id="headertwo">` is fine, but the inner div doesn't do anything so can be removed.